cmake/archdetect.cmake

```diff
@@ -43,15 +45,14 @@ else()
     message(STATUS "Arch detected: '${ARCH}'")
 endif()
 
-
```

Comment:
Mixing code and white-space changes is a no-go...

Reply:
I understand that from before. Personally I am not so strict about it.

Reply:
I'm just reminding...